tiny-postgresql-event-store/src/main/kotlin/jp/veka/converter/JsonEntityConverter.kt

```diff
@@ -0,0 +1,15 @@
+package jp.veka.converter
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import kotlin.reflect.KClass
+
+class JsonEntityConverter : EntityConverter {
+    private val objectMapper = jacksonObjectMapper()
```

Comment:
everything should be injectable so that user can exchange the implementation on a fly. So `objectMapper` implementation should be passed to JsonEntityConverter constructor